src/test/java/org/folio/fqm/service/QueryManagementServiceTest.java

```diff
@@ -242,106 +242,6 @@ void shouldReturnQueryDetailsWithContents() {
     assertEquals(expectedDetails, actualDetails);
   }
 
-  @Test
-  void shouldReturnQueryContentsWithOnlyIdsForEmptyFields() {
```

Comment:
This test (and all the other removed ones) were used to ensure that id column logic was handled correctly even if idColumns weren't in the query's list of fields. But since we're now checking that during the POST, these are no longer needed.